src/Rules/Keywords/RequireFileExistsRule.php

```diff
@@ -0,0 +1,132 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Keywords;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Include_;
+use PHPStan\Analyser\Scope;
+use PHPStan\File\FileHelper;
+use PHPStan\Rules\IdentifierRuleError;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\ShouldNotHappenException;
+use function dirname;
+use function explode;
+use function get_include_path;
+use function getcwd;
+use function is_file;
+use function is_string;
+use function sprintf;
+use function str_replace;
+use const PATH_SEPARATOR;
+
+/**
+ * @implements Rule<Include_>
+ */
+final class RequireFileExistsRule implements Rule
+{
+
+	public function getNodeType(): string
+	{
+		return Include_::class;
+	}
+
+	public function processNode(Node $node, Scope $scope): array
+	{
+		$errors = [];
+		$paths = $this->resolveFilePaths($node, $scope);
+
+		foreach ($paths as $path) {
+			if ($this->doesFileExist($path, $scope)) {
+				continue;
+			}
+
+			$errors[] = $this->getErrorMessage($node, $path);
+		}
+
+		return $errors;
+	}
+
+	/**
+	 * We cannot use `stream_resolve_include_path` as it works based on the calling script.
+	 * This method simulates the behavior of `stream_resolve_include_path` but for the given scope.
+	 * The priority order is the following:
+	 * 	1. The current working directory.
+	 * 	2. The include path.
+	 *  3. The path of the script that is being executed.
+	 */
+	private function doesFileExist(string $path, Scope $scope): bool
+	{
+		$directories = [
+			getcwd(),
```

Comment:
Inject `$currentWorkingDirectory` into the constructor. Look for `%currentWorkingDirectory%` in `conf/` to see how it's done.

Reply:
Fixed in https://github.com/phpstan/phpstan-src/pull/3294/commits/67ee4d818e82eff2adf5b509654502a1e918721d